### DISCUSSION TITLE
[SOLVED] filename length too short - disabling preview pane?

### DISCUSSION THREAD

**Discussion by marcusfoenix** [gokcehan/lf#1401] [General]
hello all,

Thanks for this great software, I have replaced ranger with this one for a few weeks now, very happy, love the bar style.

regarding my issue, i'm out of google-results :/

how do you disable the default preview pane (ie: third column/panel)? I have some lengthy filename and unfortunately  even if there's no preview (content binary for instance), the middle column/panel remains too tight and trim file names..

Sure, there's the top line which displays the filename but somehow, I don't have the reflex to look at it - a bit like preview  line on a mobile on-display keyboard. :) 
Anyway I'd prefer, if you don't mind, to see the whole file list and filter out on the go.

many thanks

**Comment by ilyagr** [comment]
You just need to do `:set preview false`. See the docs (e.g. `:doc`) for more details.

You can also put `map zp :set preview!` in your `lfrc` and then use `zp` to toggle the preview on and off.

**Comment by marcusfoenix** [comment]
hello ilyagr,

Setting preview to false was the first thing I ever did.
The 3rd panel/column remains and filenames are still truncated.

Though, I never tested the toggle syntax

Seems to be a bug:
set preview false ... does nothing from the configuration file and at at runtime
set preview! ........... toggling works
set preview true .... works at runtime 


thank you ilyagr

**Reply by ilyagr** [reply]
If you have an older lf version, then the commands should be `set preview`, `set nopreview`, and `set preview!`.

**Reply by joelim-work** [reply]
@marcusfoenix You might also want to have a look at the `ratios` setting, this configures the number of panes and their sizes. The `preview` option simply determines whether the rightmost pane is used for previewing or used as a normal pane.

There are also examples in Tips section of the wiki, for example [toggling the preview column](https://github.com/gokcehan/lf/wiki/Tips#toggling-the-preview-column).

**Reply by marcusfoenix** [reply]
thank you @joelim-work, the combination of the two solves my case. Yes , i know, filenames going up to 80 chars might sound weird.
 :)

**Comment by marcusfoenix** [comment]
I used these two versions:
- lf@v0.0.0-20220626123617-aee5c3371fdb   (I rolled back, the one I use right now)
- lf@v0.0.0-20230828013016-63d21b5a3e2d    

Regarding my issue, i'm OK with the toggling syntax . I close the discussion.

huge thank you!
